### PR TITLE
Added math.Vector2 subclass test for issue #552

### PR DESCRIPTION
**Pull request by cmtrapp02**
Added new function that tests a subclassed math.Vector2 for issue #552.
This is my first contribution on Github, so any feedback if necessary would be greatly appreciated!

### CODE REVIEW

**Comment by illume** [comment]
Thank you!

File: test/math_test.py [hunk 1]
```diff
@@ -1577,6 +1577,13 @@ def test_pickle(self):
         self.assertEqual(pickle.loads(pickle.dumps(v2)), v2)
         self.assertEqual(pickle.loads(pickle.dumps(v3)), v3)
 
+    def testVector2Subclass(self):
+        class Vector(pygame.math.Vector2):
+            pass
+        v = Vector(2.0,2.0)
```
Comment (illume) [inline test/math_test.py]: It's usual to put a space after arguments. For example: `v = Vector(2.0, 2.0)`

Comment (cmtrapp02) [reply test/math_test.py]: Okay thanks, I'll be sure to do that next time.

File: test/math_test.py [hunk 2]
```diff
@@ -1577,6 +1577,13 @@ def test_pickle(self):
         self.assertEqual(pickle.loads(pickle.dumps(v2)), v2)
         self.assertEqual(pickle.loads(pickle.dumps(v3)), v3)
 
+    def testVector2Subclass(self):
```
Comment (illume) [inline test/math_test.py]: Can you please add a decorator to show that this is an expected failure?
Please see the documentation here on how to do that: https://docs.python.org/3/library/unittest.html#unittest.expectedFailure

This way the tests will pass on TravisCI and Appveyor.

File: test/math_test.py (hunk 1 above)
Comment (illume) [reply test/math_test.py]: The pygame code isn't the cleanest yet, or using modern conventions everywhere. But eventually we will move to using [pylint](https://www.pylint.org/) to help catch these types of issues.

File: test/math_test.py (hunk 2 above)
Comment (cmtrapp02) [reply test/math_test.py]: Yup, I'll do that as soon as I can. Thanks for the help

**Comment by Nunu-Willump** [comment]
Wouldn't it be desirable to test this in a child process? The way it is now the test doesn't really fail, but instead the process just terminates with only the message "Segmentation fault (core dumped)"

**Comment by illume** [comment]
Wonderful. Thanks :)

**Comment by illume** [comment]
Ah yes. The CI (Appveyor and Travis) are failing because it crashes python. I'll merge this in now anyway, and merge the other one right away.

**Pull request merged**